freetextresponse/private/view.js

```diff
@@ -57,12 +63,19 @@ function FreeTextResponseView(runtime, element) {
                 buttonSubmit.text(buttonSubmit[0].dataset.value);
                 userAlertMessage.text(response.user_alert);
                 buttonSave.addClass(response.nodisplay_class);
-                setClassForTextAreaParent(response.indicator_class); 
+                setClassForTextAreaParent(response.indicator_class);
+                if (!response.user_alert && response.display_other_responses) {
```

Comment:
We check the user alert because submit still goes through if there are issues like invalid word count. I can also do this in the python- will that be more logical?